doc/man3/RAND_add.pod

```diff
@@ -71,6 +69,54 @@ RAND_event() returns RAND_status().
 
 The other functions do not return values.
 
+=head1 NOTES
+
+The new OpenSSL DRBG has some peculiarities which need to be taken
+into account when it is selected as the default OpenSSL CSPRNG, i.e.,
+when RAND_get_rand_method(void) == RAND_OpenSSL(). This applies in
+particular to the way reseeding is done by the DRBG:
+
+ * The DRBG seeds itself automatically, pulling randomness from trusted
+   entropy sources. Automatic reseeding occurs after 2^24 generate requests.
+   The selection of the trusted entropy sources is configured at build
+   time using the --with-rand-seed option.
+
+ * The DRBG distinguishes two different types of randomness input:
+   'entropy', which comes from a trusted source, and 'additional input',
+   which can optionally be added by the user and is considered untrusted.
+
+Automatic seeding can be disabled using the --with-rand-seed=none option.
+
+=head2 DRBG with automatic seeding enabled
+
+Calling RAND_poll() or RAND_add() is not  necessary, because the DRBG
+polls the entropy source automatically. But it is allowed to do it:
+
+Calling RAND_poll() results in a reseeding of the default DRBG.
+
+RAND_add() can be used to add both kinds of randomness, depending on the
+value of the B<randomness> argument:
+
+  randomness == 0:  The random bytes are mixed as additional input into the
+	current state of the DRBG. This is only possible if the DRBG is already
+	instantiated. Mixing in additional input is not considered a full
+	reseeding, hence the reseed counter is not reset.
+
+  randomness > 0:   The random bytes are used as entropy input for a full
+	reseeding (resp. reinstantiation) if the DRBG is already instantiated
+	(resp. uninstantiated or in an error state). A reseeding requires
+	16 bytes (128 bits) of randomness. It is possible to provide less
+	randomness than required. In this case the remaining randomness will
+	be pulled automatically from the trusted entropy sources.
```

Comment:
Reword: "In this case the missing randomness will be obtained by pulling random bytes from the trusted entropy sources."